cmd/maintained-apps/validate/main.go

```diff
@@ -0,0 +1,368 @@
+//go:build darwin || windows
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"slices"
+	"strings"
+	"time"
+
+	maintained_apps "github.com/fleetdm/fleet/v4/ee/maintained-apps"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	mdm_maintained_apps "github.com/fleetdm/fleet/v4/server/mdm/maintainedapps"
+	kitlog "github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+)
+
+var (
```

Comment:
nit: These might be better as fields on structs or passed as arguments, vs global variables. I think
this isn't blocking for rn because this is a relatively small program, but maybe as a follow up
later on.

Reply:
I'll take the config approach I have seen in a few places in the app.
```
type Config struct {
	tmpDir                      string
	env                         []string
	installationSearchDirectory string
	operatingSystem             string
	logger                      kitlog.Logger
	logLevel                    string
}
func main() {
      cfg := &Config{}
      ....
}
func run(cfg *Config) error { ... }
```

This will actually improve some of the tests I have no because I just stub a global variable. 👍